src/metabase/server/middleware/session.clj

```diff
@@ -51,6 +52,15 @@
   [response]
   (reduce clear-cookie (wrap-body-if-needed response) [metabase-session-cookie metabase-embedded-session-cookie]))
 
+(defn- use-permanent-cookies?
```

Comment:
This might be a little misleading, since the cookies aren't permanent. They last 2 weeks or for however long someone configures session length. Maybe `use-persistent-cookies?` would be a better name?